modules/editable-layers/package.json

```diff
@@ -1,9 +1,8 @@
 {
-  "private": true,
```

Comment:
You'll also need to add:

publishConfig: {access: "public"}

Without it, npm assumes a scoped package is private by default the first time it's published, and requires payment. After that the publishConfig can be removed.

Reply:
Thanks, that got me one step closer - but now I have another cryptic error:

```
npm ERR! 404 Not Found - PUT https://registry.npmjs.org/@deck.gl-community%2feditable-layers - Not found
npm ERR! 404
npm ERR! 404  '@deck.gl-community/editable-layers@9.0.0-alpha.1' is not in this registry.
```

I know it's not in the registry, that's why I'm trying to publish! 🙄

Reply:
`npm publish --access public` also no help, following https://docs.npmjs.com/creating-and-publishing-scoped-public-packages#publishing-scoped-public-packages

Reply:
That error is new to me, but looks a lot like this:

https://stackoverflow.com/q/39115101

Possibly you don't have have permission to publish to the `@deck.gl-community` scope and it's just giving an opaque error message, @ibgreen would you be able to check the npm permissions?